.azure-pipelines/install_deps.yml

```diff
@@ -14,10 +14,11 @@ steps:
     - task: GoTool@0
       inputs:
         version: $(GOVER)
-        goPath:  $(GOPATH)
+        goPath: $(GOPATH)
+        goBin: $(GOBIN)
       displayName: Install GoLang
     - script: |
-        go get -u github.com/cucumber/godog/cmd/godog
+        GO111MODULE=on go get github.com/cucumber/godog/cmd/godog@v0.8.0
```

Comment:
You have to install it using modules now, they merged breaking changes into each of their old branches, so using the old `go get` can't build the binary anymore